ingestion-beam/src/main/java/com/mozilla/telemetry/contextualservices/ParseReportingUrl.java

```diff
@@ -295,6 +295,7 @@ private static void addAdditionalDimensionsForTopSitesClicks(BuildReportingUrl b
     }
 
     if (ipReputation != null && ipReputation < IP_REPUTATION_THRESHOLD) {
+      PerDocTypeCounter.inc(attributes, "rejected_low_ip_reputation");
```

Comment:
Its good to have this metric here but i'm fairly certain we have turned off ip reputation checking as it was expensive/slow and not providing much value.

Reply:
I say leave it in to verify.